v0/destinations/candu/transform.js

```diff
@@ -14,9 +14,12 @@ const responseBuilder = (body, { Config }) => {
   payload.context = {
     source: "RudderStack"
   };
+  if (payload.userId) {
+    payload.userId = `${payload.userId}`;
```

Comment:
Are there any other fields apart from `userId` with this kind of data type requirement? Also, reach out to the Candu team if needed.

Reply:
I have checked by sending some calls through, but don't think there is any other field that has such requirements. We can ask the Candu team once for verification. WDYT?

Reply:
Yes. Let's validate.

Reply:
I checked with Candu team, they have confirmed there is not any other field with such data type requirements.